src/components/icons/IconWrapper.tsx

```diff
@@ -34,34 +34,17 @@ function IconBase(props: Readonly<IconProps & { children: React.ReactNode }>) {
 
 export const IconMaterialWrapper = styled(IconBase)`
   font-family: 'Material Symbols Rounded';
-  font-size: 1.2em;
-  font-weight: normal;
+  font-size: calc(1em * (18 / 14));
   font-style: normal;
-  line-height: 1;
-  letter-spacing: normal;
-  text-transform: none;
-  white-space: nowrap;
-  word-wrap: normal;
-  direction: ltr;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-rendering: optimizeLegibility;
+  font-weight: normal;
+  height: calc(1em * (20 / 18));
+  line-height: calc(1em * (20 / 18));
+  text-align: center;
+  width: calc(1em * (20 / 18));
```

Comment:
this works with display: inline ??

Reply:
no, `inline-block` is needed. And as I mentioned, making it `vertical-align: bottom` aligns it without the need for a flex container

Reply:
Whoops, still missing `display: inline-block`